app/frontend/src/cljs/teet/ui/common.cljs

```diff
@@ -592,3 +593,20 @@
         [Paper
          (mapc (r/partial context-menu-item toggle!)
                (remove nil? items))]]]]]))
+
+(defn date-label-component
+  "Shows :label with formatted date from :value with an info icon and :tooltip"
+  [{value :value label :label tooltip :tooltip}]
+  [Grid
+   {:container true
+    :data-cy "date-label-component"} label
+   [Grid
+    {:container true
+     :align-items :center}
+    [popper-tooltip tooltip
+     [icons/action-info-outlined
+      {:style {:color theme-colors/blue-tab}}]]
```

Comment:
Icons can be given an option :color :primary to make them use the primary color selected in the theme templates. Is this icon supposed to have some other color than the primary one?

Reply:
This color theme-colors/blue-tab is not the same as :primary This was specific deisgn requirement to have this color applied to the icon.